sechub-web-ui/src/types/global.d.ts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+declare module 'insane';
+
+interface ImportMeta {
+  env: {
+    SECHUB_API_HOST: string
```

Comment:
I think `SECHUB_HOST` is better
or just `HOST`

keeping your names short wherever possible is always more readable :)